### ISSUE TITLE
Logs should not be dumped in $(pwd)/logs

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#12277]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Linux, Godot master branch (8bc96cc14)

**Issue description:**
Since the addition of rotated logs by @endragor, everytime you run Godot a log is written to a `logs` subfolder in the current directory. It's very inconvenient when you have Godot in your $PATH and are used to running it from various places, as it litters your filesystem with log files.

I suggest that log files be written to `~/.godot/logs/` on Linux, and the respective Godot user folders on other platforms.

**Comment by CowThing** [comment]
Is there a way to turn the logs off for a game? Or should there be an option to?

**Comment by akien-mga** [comment]
If it's not the case already, it should be made optional for exported games, yes. And the path should likely also be made configurable, as some will want their logs in `user://` or in the game's folder.

**Comment by endragor** [comment]
In fact it's already uses `user://` path, the problem is that `user://` scheme is not set to the proper folder at the time the logger is initialized. I'll look into it.

**Comment by akien-mga** [comment]
I've pushed a hack (87694d188) to disable the RotatedFileLogger for now, as I'm about to release alpha2 and users will be quite confused to end up with `logs/` folder everywhere on their system (at least I have one in all my project folders, as well as all unrelated folders where I happened to be while running godot from the $PATH, e.g. `~`, `~/tmp`, `~/Projects/godot`, etc.

That hack can be reverted once a proper fix for this issue is available.
